runtime/hydradx/src/assets.rs

```diff
@@ -299,7 +300,7 @@ impl pallet_ema_oracle::Config for Runtime {
 	type SupportedPeriods = SupportedPeriods;
 	/// With every asset trading against LRNA we will only have as many pairs as there will be assets, so
 	/// 20 seems a decent upper bound for the forseeable future.
-	type MaxUniqueEntries = ConstU32<20>;
+	type MaxUniqueEntries = ConstU32<40>;
```

Comment:
adjust comment

Reply:
what is this change based on? do we have enough weight for that?

Reply:
It's based on the UI showing 16 assets (including LRNA) in the omnipool. Currently all pairs (including the new stableswap pairs) end up in the same accumulator and share this MaxEntries config. With a stablepool of 4 or 2 stablepools of 3 we would hit the cap already.
The weights should be fine if the benchmarks are rerun, just means fewer trades per block (and slightly higher fees).